api/endpoint/domain.go

```diff
@@ -27,7 +35,7 @@ func GetAddressByCoinAndDomain(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, model.CreateErrorResponse(model.InvalidQuery, err))
 		return
 	}
-	result, err := domains.HandleLookup(name, []uint64{coin})
+	result, err := domainsService.HandleLookup(name, []uint64{coin})
```

Comment:
domainsService is not a good naming, we need to encapsulate all the service logic from implementations

Reply:
I intentionally not used simply 'domains', because that's the package name, and that way accidentally direct usage can happen.  What name would you suggest?